CMakeLists.txt

```diff
@@ -63,6 +63,13 @@ if(APPLE)
   endif()
 endif()
 
+# Get the relative path from the tomviz executable install location
+# (INSTALL_RUNTIME_DIR) to the python install dir (tomviz_python_install_dir).
+# Tomviz will use this when it starts to find the python packages.
+cmake_path(RELATIVE_PATH tomviz_python_install_dir
+           BASE_DIRECTORY "${INSTALL_RUNTIME_DIR}"
+           OUTPUT_VARIABLE tomviz_path_from_exe_to_python_install_dir)
```

Comment:
`file(RELATIVE_PATH)` is another way to do this that doesn't require CMake 3.20.

Reply:
@psavery Can you fix this up so we don't need 2.30?